ros2_batch_job/__main__.py

```diff
@@ -377,11 +377,13 @@ def build_and_test(args, job):
 
     print('# BEGIN SUBSECTION: test')
 
+    # xunit2 format is needed to make Jenkins xunit plugin 2.x happy
     test_cmd = [
         args.colcon_script, 'test',
         '--base-paths', '"%s"' % args.sourcespace,
         '--build-base', '"%s"' % args.buildspace,
         '--install-base', '"%s"' % args.installspace,
+        '--pytest-args', '-o junit_family=xunit2',
```

Comment:
How does this behave when the user chooses to pass custom `--pytest-args`?

Reply:
During my tests colcon ended up with two `--pytest-args` if one was supplied as user input parameter. This does not work aggregating options so it was not a valid solution. I changed the code to use a [ini file with the configuration](https://docs.pytest.org/en/latest/deprecations.html#junit-family-default-value-change-to-xunit2). Tests are happy.